### ISSUE TITLE
'use strict' is duplicated in transpiled output.

### ISSUE THREAD

**Issue by STRML** [babel/babel#1538]
I am working on finding the version where this broke.

The following input:

```javascript
'use strict';

import * as React from 'react-tools/build/modules/React';

module.exports = React.createClass({
  render: function(){
    return <div></div>;
  }
});
```

Should produce:

```javascript
'use strict';

var _interopRequireWildcard = function (obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (typeof obj === 'object' && obj !== null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj['default'] = obj; return newObj; } };

var _import = require('react-tools/build/modules/React');

var React = _interopRequireWildcard(_import);

module.exports = React.createClass({
  displayName: 'exports',

  render: function render() {
    return React.createElement('div', null);
  }
});
```

And it does as of 5.1.13.

However, 5.3.3 produces:

```javascript
'use strict';

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) newObj[key] = obj[key]; } } newObj['default'] = obj; return newObj; } }

var _reactToolsBuildModulesReact = require('react-tools/build/modules/React');

var React = _interopRequireWildcard(_reactToolsBuildModulesReact);

'use strict';

module.exports = React.createClass({
  displayName: 'exports',

  render: function render() {
    return React.createElement('div', null);
  }
});
```

Note the extra `'use strict';` on line 9.

**Comment by sebmck** [comment]
Fixed, this would have happened right on verison 5.3.0. Also FWIW, you don't need to have the `"use strict";` directives in your code, Babel automatically adds it.

**Comment by STRML** [comment]
Thanks!

We have been using them for some time in test cases for JSXHint to cover
all compilers (jstransform, babel) and test cases started failing all of
a sudden. Thank you for the quick fix.

**Comment by sebmck** [comment]
Fixed as of 5.4.0, thanks!
